PreMailer.Net/PreMailer.Net.Tests/LinkTagCssSourceTests.cs

```diff
@@ -61,9 +61,9 @@ public void GetCSS_CallsWebDownloader_WithSpecifiedBundle()
 		[TestMethod]
 		public void GetCSS_AbsoluteUrlInHref_CallsWebDownloader_WithSpecifiedPath()
 		{
-			string path = "http://b.co/a.css";
+			var path = "http://b.co/a.css";
 
-			LinkTagCssSource sut = CreateSUT(path: path);
+			var sut = CreateSUT(path: path);
```

Comment:
I'm a little torn in a case like this.

As you write the code, you probably know exactly what you "have in hand", so explicit type declaration won't make much of a difference.

But when you open up this in three months, you won't remeber what `CreateSUT` returned here, but you may still recall what the `LinkTagCssSource` has to offer.

I worked with a guy once, who said you should be able to tell the type by reading the line.

So `var path = "http://b.co";` is just as good as `string path = "http://b.co";`.

But there's a big difference in readability on these:
```csharp
var sut = CreateSUT();
LinkTagCssSource sut = CreateSUT():
```

When instatiating, it is fine to use `var` as the type is written:
```csharp
var cssSource = new LinkTagCssSource();
```